packages/blade/src/components/Card/Divider.native.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components/native';
+import { makeSize } from '~utils';
+
+const Divider = styled.View(({ theme }) => {
```

Comment:
can we make divider a generic component and move it out?

Reply:
Let's wait for more usecase? Does action list have dividers? Maybe @saurabhdaware can extract this out in that case.

Reply:
yes action list does have divider. sure, we can scope it here only but lets raise this to design to build this as a component and we can extract this out then

Reply:
will do.